locale/en/blog/vulnerability/openssl-november-2022.md

```diff
@@ -23,8 +23,11 @@ the _highest_ severity issue is CRITICAL.
 
 [security policy](https://www.openssl.org/policies/secpolicy.html).
 
-Node.js v16.x, v18.x and v19.x use OpenSSL v3.
-Therefore all active LTS release lines are impacted by this update.
+Node.js v18.x and v19.x use OpenSSL v3.
+Therefore these active LTS release lines are impacted by this update.
+
+Node.js v16.x uses OpenSSL v1.1.1.
+Therefore it is currently NOT expected to be impacted by this update.
```

Comment:
```suggestion
```

Reply:
Is this considered a dupe of https://github.com/nodejs/nodejs.org/pull/4901?